misc/python/materialize/mzbuild.py

```diff
@@ -256,7 +256,11 @@ def generate_cargo_build_command(
         bins: list[str],
         examples: list[str],
     ) -> list[str]:
-        rustflags = rustc_flags.coverage if rd.coverage else ["--cfg=tokio_unstable"]
+        rustflags = (
+            rustc_flags.coverage
+            if rd.coverage
+            else ["--cfg=tokio_unstable", "--remap-path-prefix", f"{os.getcwd()}/="]
```

Comment:
This seems more likely to work, although I do wonder if you want `ROOT` rather than `os.getcwd()` as the FROM.